static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -56,6 +56,14 @@ const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
             const adSlot = iframe && iframe.closest('.js-ad-slot');
+            if (adSlot.classList.contains('ad-slot--mostpop')) {
+                // We ignore resize events (sent mainly by apnx)
+                // for the mostpop (Most popular) slot
+                // See https://trello.com/c/TtuGq6Iy
+                return Promise.reject(
+                    new Error('No resizing for ad-slot--mostpop')
```

Comment:
Do we need to throw an error here and if so where do we handle it?

If we are aiming to ignore the resize on mostpop, could we just build in an early return instead?

```js
if (adSlot.classList && adSlot.classList.contains('ad-slot--mostpop')) {
    return Promise.resolve(false);
}
```

Resolving false is optional 🙃